src/components/Savings/Stats/Stats.tsx

```diff
@@ -52,45 +91,44 @@ const Stats = () => {
               <Icon variant="info" size="tiny" color="var(--textAccentInverted)" />
             </TooltipWrapper>
           </div>
-          <div className="raft__savings__stats__item-token">
```

Comment:
This class `raft__savings__stats__item-token` is no longer used in any component, remove it from .scss file as well.